src/Webcreate/Conveyor/Transporter/SftpTransporter.php

```diff
@@ -101,7 +100,7 @@ public function login()
             }
 
             $identityFile = file_get_contents($identityFilePath);
-            $key          = new \Crypt_RSA();
+            $key          = new RSA();
             $loaded       = $key->loadKey($identityFile);
```

Comment:
Hm, it seems the change is in master, so it will probably be in a next major release, I think:

https://github.com/phpseclib/phpseclib/commit/ec3fe7277b8b1c4250c939f39783840ca12c1a19